gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_2/00_add_event_organizations.yml

```diff
@@ -6,13 +6,13 @@ databaseChangeLog:
         - createTable:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.